Godeps/Godeps.json

```diff
@@ -345,6 +340,10 @@
 			"Rev": "f7445b17d61953e333441674c2d11e91ae4559d3"
 		},
 		{
+			"ImportPath": "golang.org/x/net/websocket",
+			"Rev": "d9558e5c97f85372afee28cf2b6059d7d3818919"
```

Comment:
I couldn't match up the SHAs between hosts, so I just vendored from master and looked at the diff. I agree, it looks like safe changes that logspout probably wouldn't care about anyway. :four_leaf_clover: